docs/source/en/tasks/translation.md

```diff
@@ -348,7 +348,8 @@ The simplest way to try out your finetuned model for inference is to use it in a
 ```py
 >>> from transformers import pipeline
 
->>> translator = pipeline("translation", model="my_awesome_opus_books_model")
+# Change `xx` to the language of the input and `yy` to the language of the desired output.
```

Comment:
we could mention what format these langauge should be, `en` ? `english` `eng` , pointing to a list! Otherwise good